src/widgets/stories/constants/multiInputSchemas.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+export const schema = {
+  type: 'object',
+  title: '',
+  properties: {
+    multipleInput1: {
+      type: 'array',
+      title: 'Multiple input (ui:options flexOptions: [1, 1])',
+      items: {
+        type: 'object',
+        enum: ['Question 1', 'Question 2', 'Question 3']
+      },
+      minItems: 1,
+      uniqueItems: true
+    },
+    multipleInput2: {
+      type: 'array',
+      title: 'Multiple input (ui:options flexOptions: [2, 1])',
+      items: {
+        type: 'object',
+        enum: ['Question 1', 'Question 2', 'Question 3']
+      },
+      minItems: 1,
+      uniqueItems: true
+    },
+    inputTypeNumber: {
+      type: 'array',
+      title: 'Multiple input (ui:options inputType: number, flexOptions: [4, 1])',
+      items: {
+        type: 'number',
+        enum: ['Question 1', 'Question 2', 'Question 3']
```

Comment:
I feel like there's a discrepancy here where the type is 'number' but the enums are not. Typically `enum` holds the potential values where as `enumNames` adjusts their labels.

Reply:
For the purposes of this widget I'm using `enum` to denote the labels of the form inputs and not the potential values. I know it's an abuse of terminology but I don't see any other way.

Reply:
since the `enum` values don't actually matter, the schema could have

enum: [1,2,3]
enumNames: ['Question 1', 'Question 2', 'Question 3']
and the widget would just use the enumNames to render.